tiles/src/main/java/com/protomaps/basemap/layers/Roads.java

```diff
@@ -146,12 +176,14 @@ public void processFeature(SourceFeature sourceFeature, FeatureCollector feature
 
   @Override
   public List<VectorTile.Feature> postProcess(int zoom, List<VectorTile.Feature> items) {
+    if( zoom < 12 ) {
```

Comment:
The linkSimplify logic is still mysterious to me... either it has a bug or it shouldn't be exercised at higher zooms (or both!?), else links were totally missing at mid-zooms

Reply:
It produces gaps at mid-zooms still